pkg/scd/operations_handler.go

```diff
@@ -224,6 +225,19 @@ func (a *Server) PutOperationReference(ctx context.Context, req *scdpb.PutOperat
 	}, key)
 
 	if err != nil {
+		if err == scderr.MissingOVNsInternalError() {
+			// The client is missing some OVNs; provide the pointers to the
+			// information they need
+			ops, err := a.Store.SearchOperations(ctx, uExtent, owner)
+			if err != nil {
+				return nil, err
+			}
+			intendedErr, unintentionalErr := scderr.MissingOVNsErrorResponse(ops)
```

Comment:
nit: double errors are a little off-putting. Might be nice to do one error, and check if it's not a MissingOVNsError. Or could pass an error pointer and return an error, or could return a string to create a new error.

Reply:
Will change.

Reply:
Done.